server/routes/auth-routes.js

```diff
@@ -30,11 +35,16 @@ router.get(
 );
 
 // Google OAuth
-router.get("/google", passport.authenticate("google",{ scope: ['https://www.googleapis.com/auth/plus.login'] }));
+router.get(
+  "/google",
+  passport.authenticate("google", {
+    scope: ["https://www.googleapis.com/auth/plus.login"]
+  })
+);
 
 router.get(
   "/google/callback",
-  passport.authenticate("google", { failureRedirect: "/login" }),
+  passport.authenticate("google", { failureRedirect: "/login" }, generateToken),
```

Comment:
Not able to sign in using Google, I think the genrateToken method has been misplaced.